db/seeds/board-seeds.js

```diff
@@ -1,53 +1,53 @@
-const { User, Board } = require('../models');
+const { User, Board } = require('../../models');
 
 const boarddata = [
   {
     user_id: 5,
     title: 'Far far away',
     description: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit.Aenean commodo ligula eget dolor.Aenean massa.Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem.Nulla consequat massa quis enim.Donec pede justo, fringilla vel, aliquet nec, vulputate eget, arcu.In enim justo, rhoncus ut, imperdiet a, venenatis vitae, justo.Nullam dictum felis eu pede mollis pretium.Integer tincidunt.Cras dapibu',
-    // board_content: insert JSON or string here
+    board_content: '{"some":"Content"}',
```

Comment:
Is there a dummy "saved" board we can add in here instead of the image to see how this works?